src/algorithms/generate.rs

```diff
@@ -1,10 +1,8 @@
 //! Generate prime components for the RSA Private Key
 
 use alloc::vec::Vec;
-use num_bigint::{BigUint, RandPrime};
-#[allow(unused_imports)]
-use num_traits::Float;
```

Comment:
[`Float::ln`](https://docs.rs/num-traits/latest/num_traits/float/trait.Float.html#tymethod.ln) was used to provide support for f64::ln on non-std (used line 45).

Reply:
thanks, fixed now